src/Nethermind/Nethermind.Blockchain.Test/Synchronization/FastSync/NodeDataDownloaderTests.cs

```diff
@@ -316,21 +317,25 @@ private static (string Name, Action<StateTree, StateDb, StateDb> Action)[] InitS
         private StateDb _localStateDb;
         private StateTree _remoteStateTree;
         private StateTree _localStateTree;
-
-        static NodeDataDownloaderTests()
+        
+        [MethodImpl(MethodImplOptions.Synchronized)]
```

Comment:
Do you want something like https://github.com/nunit/docs/wiki/SetUpFixture-Attribute ?

Reply:
hm, yes

Reply:
no, actually, setup fixture does not work